rewrite-test/src/main/kotlin/org/openrewrite/java/JavaParserTest.kt

```diff
@@ -35,14 +36,97 @@ interface JavaParserTest : JavaRecipeTest {
            public class A {}
        """.trimIndent();
 
-        val cu = JavaParser.fromJavaVersion().build().parse(source).get(0);
+        val cu = JavaParser.fromJavaVersion().build().parse(source).get(0)
 
-        assertThat(cu.printAll()).isEqualTo(source);
+        assertThat(cu.printAll()).isEqualTo(source)
 
-        val newCu = JavaVisitor<ExecutionContext>().visit(cu, InMemoryExecutionContext()) as J.CompilationUnit;
+        val newCu = JavaVisitor<ExecutionContext>().visit(cu, InMemoryExecutionContext()) as J.CompilationUnit
 
-        assertThat(newCu.printAll()).isEqualTo(source);
+        assertThat(newCu.printAll()).isEqualTo(source)
 
     }
 
+    @Test
+    fun javaModulePrints1(jp: JavaParser) = assertUnchanged(
+        recipe = AutoFormat(),
+        before = """
+            module com.example.foo {
+                opens com.example.foo.internal to com.example.foo.network, com.example.foo.probe;
+            }
+        """
+    )
```

Comment:
I would put these into their own `ModuleTest`, using the `RewriteTest` interface rather than `JavaRecipeTest`.

`AutoFormat` has no place in these tests. Here it can only serve to confound the results, be another layer we have to debug through if there's a parsing problem. Tests of `AutoFormat`'s ability to handle modules should go in `AutoFormat`'s tests.